src/styles/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: #f3f3f3 1px solid;
+}
+
+.card__description {
+  padding: 16px;
+}
+
+.card__image {
+  padding-top: 32px;
+  display: flex;
+  justify-content: center;
+}
+
+.image__imac {
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  padding-top: 24px;
```

Comment:
[padding vs margin](https://blog.hubspot.com/website/css-margin-vs-padding#:~:text=In%20CSS%2C%20a%20margin%20is,the%20space%20inside%20an%20element.)